src/index.js

```diff
@@ -58,13 +43,36 @@ const loadIconAndEvent = () => {
   menus.forEach((img) => {
     img.src = menu;
     img.addEventListener('mouseover', () => {
-      if (img.src !== deleteIcon) img.src = menuHover;
-      else img.src = deleteIconHover;
+      if (img.src !== deleteIcon) {
+        img.src = menuHover;
+        img.classList.remove('cursor-hand');
+        img.classList.add('cursor-move');
+      } else {
+        img.src = deleteIconHover;
+        img.classList.remove('cursor-move');
+        img.classList.add('cursor-hand');
+      }
     });
     img.addEventListener('mouseout', () => {
       if (img.src !== deleteIconHover) img.src = menu;
       else img.src = deleteIcon;
     });
+    /**
+     * remove one task--------------
+     */
+    img.addEventListener('click', () => {
+      if (img.src === deleteIconHover || img.src === deleteIcon) {
+        const parent = img.parentNode.previousSibling.previousSibling;
+        const otherParent = parent.previousSibling.previousSibling;
+        const imgId = parseInt(otherParent.firstChild.nextSibling.id, 10);
+        data = Utilities.clearOneTask(data, imgId);
+        Update.save(data);
+        // const tmp = img.parentNode.parentNode;
+        container.removeChild(img.parentNode.parentNode);
+        // Update.changedState(data);
+        // console.log('block:', tmp);
+      }
+    }, false);
```

Comment:
- [ ] We can not delete a task when clicking on the trash icon, actually, the `clearOneTask` function is never called. This might be due to the fact that the if condition here is never satisfied. Let's fix that.

Reply:
Thank you for your review, the code above is working and the condition is respected but there is another event blur on the same element that makes its issue. I will fix it

Reply:
This issue is well resolved! ✔️